test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/DacFx/DacFxServiceTests.cs

```diff
@@ -839,6 +839,32 @@ public async Task ValidateStreamingJob()
             dacfxRequestContext.VerifyAll();
         }
 
+        /// <summary>
+        /// Verify Object Types Dictionary items with ObjectType Enum members
+        /// </summary>
+        /// <returns></returns>
+        [Test]
+        public void ValidateObjectTypesOptionswithEnum()
+        {
+            DeploymentOptions options = new DeploymentOptions();
+
+            // Verify the object types dictionaty should exists
+            Assert.That(options.ObjectTypesDictionary, Is.Not.Null, "Object types dictionary is empty");
+
+            // Verify that the objects dictionary has all the item from Enum
+            Assert.That(options.ObjectTypesDictionary.Count, Is.EqualTo(Enum.GetNames(typeof(ObjectType)).Length), "Object types dictionary do not have all the items from ObjectType enum");
+
+            // Verify the options in the objects dictionary exists in the ObjectType Enum
+            foreach (var objTypeRow in options.ObjectTypesDictionary)
+            {
+                // Verify the option exists in ObjectType Enum
+                Assert.That(Enum.IsDefined(typeof(ObjectType), objTypeRow.Key), Is.True, @"{0} is not an enum member", objTypeRow.Key);
```

Comment:
You can use string interpolation here!

`$"{objTypeRow.Key} is not an enum member"`